Cakefile

```diff
@@ -8,9 +8,9 @@ plugins = {
     "org.apache.cordova.file": "https://git-wip-us.apache.org/repos/asf/cordova-plugin-file.git#r1.3.3"
     "org.apache.cordova.file-transfer": "https://git-wip-us.apache.org/repos/asf/cordova-plugin-file-transfer.git#r0.5.0"
     "io.cozy.cordova-images-browser": "https://github.com/aenario/cordova-images-browser"
-    "io.cozy.jsbackgroundservice": "https://github.com/jacquarg/cordova-jsbackgroundservice"
-    "io.cozy.jsbgservice-newpicture": "https://github.com/jacquarg/cordova-jsbgservice-newpicture"
-    "io.cozy.contacts": "https://github.com/jacquarg/cordova-plugin-contacts"
+    "io.cozy.jsbackgroundservice": "https://github.com/jacquarg/cordova-jsbackgroundservice#v1.0.1"
+    "io.cozy.jsbgservice-newpicture": "https://github.com/jacquarg/cordova-jsbgservice-newpicture#v1.0.1"
+    "io.cozy.contacts": "https://github.com/jacquarg/cordova-plugin-contacts#c1.0.3"
```

Comment:
A long story for one (foreign) character letter:
Initialy, contacts plugin is a fork from the cordova ones. But now it's clear that they won't merge again, (and shouldn't) . So I put a c as cozy, to notice that (https://github.com/jacquarg/cordova-plugin-contacts/releases) in the version name.

The fact is that this plugin is a mess, and I think I should rewrite it from scratch and rename it contactsync . But for now, I just wanted to notice that it was really not the cordova-plugin-contact anymore.